index.html

```diff
@@ -92,7 +161,41 @@ <h2 class="card-head">Tonic</h2>
         <li>css</li>
         <li>javaScript</li>
       </ul>
-      <button type="submit" class="btn">See Project</button>
+      <button type="submit" class="btn" onclick="openPopup2()">See Project</button>
+      <!-- START POPUP-->
+      <div class="popup" id="popup2">
+        <div class="popup-content">
+            <div class="pop-head">
+          <h2 class="card-head">Tonic</h2>
+          <a href="javascript:void(0)" class="popup-close" onclick="closePopup2()">&times;</a>
+        </div>
+          <ul class="list">
+        <li class="brand">CANOPY</li>
+        <li><img src="img/Counter.png" alt="Counter">Back End Dev</li>
+        <li><img src="img/Counter.png" alt="Counter">2015</li>
```

Comment:
Desktop Popup:

- [ ] Kindly make sure to follow the spacing for the list displayed before the image. Also, make sure to follow the color as per the Figma design.
- [ ] Kindly make sure to follow the same height and width for the popup as per the Figma design.
- [ ] Kindly make sure to follow the button text and size as per the Figma design.
- [ ] Kindly make sure to add a blurred background when the popup is open.
- [ ] Kindly make sure that the paragraph is 3/4th of the width and the list, buttons are displayed in 1/4th part.
![2022-05-12 (9)](https://user-images.githubusercontent.com/70269507/168032361-748b9e5a-cc3a-47be-b517-5d8b406c239e.png)